src/models/AppState.js

```diff
@@ -93,6 +93,12 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
     return Okta.loc('minutes', 'login', [factorLifetimeInMinutes]);
   }
 
+  function getGracePeriodRemainingDays (gracePeriodEndDate) {
+    let endDate = new Date(gracePeriodEndDate).getTime();
```

Comment:
nit: both can be consts we dont reuse them.

Reply:
Changing this to var to be consistent with the rest of the file